trytravis.py

```diff
@@ -185,8 +218,29 @@ def _submit_changes_to_github_repo(path, url):
     return commit, committed_at
 
 
+def _get_repo_info():
+    """ Get repository information form Travis API. """
+    import requests
+
+    with requests.get(
+        'https://api.travis-ci.org/repos',
+        headers=_travis_headers()
+    ) as r:
+        if not r.ok:
+            raise RuntimeError('Fail to get repository information. '
+                               'Could not reach the Travis API '
+                               'endpoint. Additional information: '
+                               '%s' % str(r.content))
+        json = r.json()
+        repos = json["repositories"]
+        for i in range(0, len(repos)):
+            if 'trytravis' in repos[i]["name"]:
+                id = repos[i]["id"]
+                return id
+
+
 def _wait_for_travis_build(url, commit, committed_at):
-    """ Waits for a Travis build to appear with the given commit SHA """
+    """ Waits for a Travis build to appear with the given commit SHA. """
```

Comment:
Adding periods to doc-strings isn't related to this change, please revert.

Reply:
ok